src/Integration.Vsix/SonarLintDaemon/SonarLintDaemon.cs

```diff
@@ -399,7 +402,14 @@ private bool IsVerbose()
             }
             WritelnToPane($"Found {issueCount} issue(s)");
 
+            fileIssues.AddOrUpdate(path, issues, (key, value) => issues);
```

Comment:
It looks like this collection is never cleared out. We should add some cache management (as a separate ticket).